applications/dashboard/controllers/class.profilecontroller.php

```diff
@@ -135,7 +135,7 @@ public function Activity($UserReference = '', $Username = '', $UserID = '', $Pag
       $this->ActivityModel->JoinComments($Activities);
       $this->SetData('Activities', $Activities);
       if (count($Activities) > 0) {
-         $LastActivity = $Activities[0];
+         $LastActivity = array_shift(array_values($Activities));
```

Comment:
`array_shift` would also remove it from `$Activities`, which isn't what this code was doing.